api/src/db/db.service.ts

```diff
@@ -137,6 +138,56 @@ export class DbService {
         });
     }
 
+    /**
+     * Get multiple documents by ID and type
+     * @param {Uuid[]} docIds - Document IDs to be included in search.
+     * @param {DocType[]} types - Document types to be included in search
+     * @returns - Promise containing the query result
+     */
+    getDocs(docIds: Uuid[], types: DocType[]): Promise<nano.MangoResponse<unknown>> {
+        // TODO: Write unit test
```

Comment:
Remember to check.

Reply:
Updated to use nano.fetch() instead of nano.find(), and implemented document type field checking in code.
Also added a test for it.